distributed/worker.py

```diff
@@ -129,11 +129,18 @@ def __init__(self, scheduler_ip, scheduler_port, ip=None, ncores=None,
         self.memory_limit = memory_limit
         if memory_limit:
             try:
-                from zict import Buffer, File, Func
+                from zict import Buffer, File, Func, LMDB, Sieve
             except ImportError:
                 raise ImportError("Please `pip install zict` for spill-to-disk workers")
             path = os.path.join(self.local_dir, 'storage')
-            storage = Func(dumps_to_disk, loads_from_disk, File(path))
+            file_storage = Func(dumps_to_disk, loads_from_disk, File(path))
+            lmdb_path = os.path.join(self.local_dir, 'lmdb')
+            lmdb = Func(dumps_to_disk, loads_from_disk, LMDB(lmdb_path))
+            def sieve_func(k, v):
+                return 'small' if sizeof(v) < 32 * 1024 else 'large'
```

Comment:
We might want to increase the size of this.  32k might still be fairly small.  One way to think about this is "how many files is a gigabyte".  In worst case here we get 30000 files in a directory, which I'm pretty sure will negatively affect the file system.